docs/source/conf.py

```diff
@@ -3,6 +3,7 @@
 # This file only contains a selection of the most common options. For a full
 # list see the documentation:
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
+import glob
```

Comment:
glob is unused

Reply:
It will be used when the line 100 is uncommented. I don't think we should remove the import just to add it back later.

Reply:
I recommend just  remove it now and revert when it will be needed